oroi/db/management/commands/load_scrape_data.py

```diff
@@ -103,19 +103,25 @@ def extact_data(self):
                                 "interest_type", "uncategorised"
                             ),
                             description=declaration_data["description"],
-                            declared_date=sorted(
-                                list(declared_on_dates),
-                                key=lambda x: x
-                                if x is not None
-                                else datetime.datetime.min,
-                            ),
                             interest_date=fuzzy_date_parse(
                                 declaration_data.get("interest_date")
                             ),
                             source=declaration_data["source"],
                             donor=declaration_data.get("interest_from"),
                         )
 
+                        declaration.declared_date = declared_date = sorted(
```

Comment:
Think you've got an extra `declared_date`  in there.

Might be good to expand this into a full helper function now that it is sufficiently complicated, or add more comments on it.... I think I can tell what it does; exclude None values, order the dates old-new and remove any duplicates to make a single list value that can be reproduced to avoid unnecessary updates?